bulk/node_bulk_load-AssetRecord.csv

```diff
@@ -61,6 +61,7 @@ CGVEH-GLDRCP-00387,A00585,Node,1,Coastal Glider 387,COASTAL GLIDER 1000M,Teledyn
 CGVEH-GLDRCP-00388,A00583,Node,1,Coastal Glider 388,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,388,,20131223,186935,,Woods Hole Oceanographic Institution
 CGVEH-GLDRCP-00389,A00584,Node,1,Coastal Glider 389,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,389,,20131223,186935,,Woods Hole Oceanographic Institution
 CGVEH-GLDRCP-00514,,Node,1,Coastal Glider 514,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,514,,,,,Woods Hole Oceanographic Institution
+CGVEH-GLDRCP-00559,,Node,1,Coastal Profiling Glider 559,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 302392-3,559,,,,,Woods Hole Oceanographic Institution
```

Comment:
Description of equipment column should be "Coastal Glider 559". This glider is being converted to a regular coastal glider and not a profiling coastal.